libs/application/templates/accident-notification/src/fields/FieldAlertMessage/index.tsx

```diff
@@ -39,9 +40,11 @@ export const FieldAlertMessage: FC<FieldBaseProps & FieldAlertMessageProps> = ({
         message={
           <>
             <Box component="span" display="block">
```

Comment:
Should this render when the description is missing?

Reply:
It appears so. This field is build to support message that includes either description or Links (or both).